sysl2/sysl/examples/grammars/go/go.gen.sysl

```diff
@@ -18,6 +18,12 @@ CodeGenTransform:
         else t.type_ref
     )
 
+  !view GetGoField(word <: string) -> string:
+    word -> (:
+      let wordUpper = ToTitle(word)
+      out = if (wordUpper in {"ACL", "API", "ASCII", "CPU", "CSS", "DNS", "EOF", "GUID", "HTML", "HTTP", "HTTPS", "ID", "IP", "JSON", "LHS", "QPS", "RAM", "RHS", "RPC", "SLA", "SMTP", "SQL", "SSH", "TCP", "TLS", "TTL", "UDP", "UI", "UID", "UUID", "URI", "URL", "UTF8", "VM", "XML", "XMPP", "XSRF", "XSS"}) == true then wordUpper else Title(ToLower(word))
```

Comment:
feature request: ability for whoever is running codegen to override rules for field naming.

suggestion:   instead of this heuristic could we add support for optionally specifying the go field names in the input sysl file, and using them.

why? each specific application domain or business domain may have its own collection of commonly used acronyms. so its not possible to have one list of acryonyms that does the right thing for all users.  for example, in telco domain, some telco companies use FAT as an acronym for a bit of network equipment, whereas in many other contexts "fat" would be a normal word.

Reply:
Yes I agree. For the moment this is added to avoid linter complains.

Reply:
Users can define their own list of naming conventions in transform and the code will be generated as expected.

Reply:
the whole purpose of the grammar and the transform files is to be able to write your own rules.

Reply:
okay, thanks for explaining.